### PR TITLE
remove paintstore

### PR DESCRIPTION
**Pull request by domdinicola**
part 2 would remove reference of paintstore from migrations and the proper requirement

### CODE REVIEW

File: EquiTrack/locations/models.py [hunk 1]
```diff
@@ -9,11 +9,10 @@
 from django.utils.encoding import python_2_unicode_compatible
 
 from mptt.models import MPTTModel, TreeForeignKey
-from paintstore.fields import ColorPickerField
 
 logger = logging.getLogger(__name__)
 
-
+# TODO remove when remove the library
 def get_random_color():
```
Comment (reinbach) [inline EquiTrack/locations/models.py]: I believe you can remove this function completely

Comment (domdinicola) [reply EquiTrack/locations/models.py]: not yet.. it's used in location 0001 migration

File: EquiTrack/locations/migrations/0001_initial.py [hunk 2]
```diff
@@ -32,7 +31,7 @@ class Migration(migrations.Migration):
                 ('name_col', models.CharField(default=b'name', max_length=254)),
                 ('pcode_col', models.CharField(default=b'pcode', max_length=254)),
                 ('parent_code_col', models.CharField(blank=True, max_length=254, null=True)),
-                ('color', paintstore.fields.ColorPickerField(blank=True, default=locations.models.get_random_color, max_length=7, null=True)),
+                ('color', models.CharField(blank=True, default='#ffffff', max_length=7, null=True)),
```
Comment (ntrncic) [inline EquiTrack/locations/migrations/0001_initial.py]: we should leave the default `get_random_color`

File: EquiTrack/locations/models.py [hunk 3]
```diff
@@ -9,19 +8,10 @@
 from django.utils.encoding import python_2_unicode_compatible
 
 from mptt.models import MPTTModel, TreeForeignKey
-from paintstore.fields import ColorPickerField
 
 logger = logging.getLogger(__name__)
 
 
-def get_random_color():
```
Comment (ntrncic) [inline EquiTrack/locations/models.py]: leave this in

Comment (domdinicola) [reply EquiTrack/locations/models.py]: Why? Is not used elsewhere.

**Comment by domdinicola** [comment]
I don't think we don't need a migration to change it to charfield. I edit the first migration to use a charfield from the beginning (the ColorPickerField extends the charfield) so we can remove all references to paintstore in eTools (otherwise I would have to keep the requirement in or o would get an import error)

**Pull request merged**